Makefile

```diff
@@ -94,7 +94,7 @@ ifneq ($(wildcard $(ASSETS_DIR)/.),)
 	cp -r $(ASSETS_DIR) dist/$(PLUGIN_ID)/
 endif
 ifneq ($(HAS_PUBLIC),)
-	cp -r public/ dist/$(PLUGIN_ID)/
+	cp -r public/ dist/$(PLUGIN_ID)/public
```

Comment:
@agarciamontoro Without this change, the script was placing the contents of the `public` folder in the `dist/$(PLUGIN_ID)` folder, instead of placing it in a new `public` subdirectory. This was making it so those files are not accessible through the `public` endpoint.